### ISSUE TITLE
Talkback is not announcing Selected or deselected for work, personal, health, family and friends buttons in tags section: A11y_.NET 9 template_Project_Screen reader.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26940]
### GitHub Tags:
#A11yTCS; #A11ySev2; #Android; #WCAG1.3.1; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #A11yMAS; #Screen reader; #Element:Buttons; #Device; #info and Relationship; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback Version:15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till project section, and double tap on any project, Project screen will open, 
4. Swipe till category and choose a category from the list of category section.
5. Swipe till tags and choose a tag from the list of tags or user can select the task from the existing task list items.
8. Swipe till new task button in the project screen.
9. Swipe till buttons like work, personal, health, family and friends buttons in tags section and Observe that Talkback is not announcing Selected or deselected for work, personal, health, family and friends buttons in tags section.

### Actual Result:
Talkback is not announcing Selected or deselected for work, personal, health, family and friends buttons in tags section.

### Expected Result:
Talkback should be announcing Selected or deselected for work, personal, health, family and friends buttons in tags section, so that users will get know that buttons got selected or deselected.

### User Impact:
Users who relay on Screen reader will get confused that those buttons are selected or not selected.

### Attachment:
https://github.com/user-attachments/assets/1705ba9e-f738-4a29-a72a-d2eef40addd2

**Comment by jfversluis** [comment]
Maybe fixed by #26923 need to try

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

https://github.com/user-attachments/assets/a615e289-abe7-4820-91af-fbca75dcaeea
